lib/solidus_support.rb

```diff
@@ -31,28 +31,15 @@ def combined_first_and_last_name_in_address?
       Spree::Config.use_combined_first_and_last_name_in_address
     end
 
-    def new_gateway_code?
-      first_version_with_new_gateway_code = Gem::Requirement.new('>= 2.3')
-      first_version_with_new_gateway_code.satisfied_by?(Spree.solidus_gem_version)
-    end
-
     def payment_source_parent_class
-      if new_gateway_code?
-        Spree::PaymentSource
-      else
-        Spree::Base
-      end
```

Comment:
Maybe we can also deprecate `payment_source_parent_class` requesting users to just use `Spree::PaymentSource`? Same for the other method!